apps/src/sites/code.org/pages/public/teacher-dashboard/index.js

```diff
@@ -52,28 +53,13 @@ function renderSectionProjects(sectionId) {
 }
 
 function renderSectionProgress(section, validScripts) {
-  registerReducers({sectionProgress});
+  registerReducers({sectionProgress, scriptSelection});
   const store = getStore();
   store.dispatch(setSection(section));
-
-  const promises = [
-    $.ajax({
-      method: 'GET',
-      url: `/dashboardapi/sections/${section.id}/student_script_ids`,
-      dataType: 'json'
-    }),
-    $.ajax({
-      method: 'GET',
-      url: `/dashboardapi/courses?allVersions=1`,
-      dataType: 'json'
-    })
-  ];
-  Promise.all(promises).then(data => {
-    let [studentScriptsData, validCourses] = data;
-    const { studentScriptIds } = studentScriptsData;
-    store.dispatch(setValidScripts(validScripts, studentScriptIds, validCourses, section.course_id));
+  // add callback to this function?
```

Comment:
Is this a TODO or a question posed to reviewers?

Reply:
If it was a question to reviewers, it recieved an answer from Brad in a previous PR comment on a different line, which read:
```js
// In renderSectionProgress
store.dispatch(loadValidScripts(section, validScripts))
  .then(() => renderSectionProgressReact(store));
```